cmd/gossamer/configcmd_test.go

```diff
@@ -197,14 +185,11 @@ func TestGetConfig(t *testing.T) {
 		set.String(c.name, c.value, "")
 		context := cli.NewContext(app, set, nil)
 
-		fig, err := getConfig(context)
-		if err != nil {
-			t.Fatalf("failed to set fig %v", err)
-		}
+		defaultConfig, err := getConfig(context)
+		require.Nil(t, err)
+
+		require.Equal(t, defaultConfig, c.expected)
```

Comment:
shouldn't this be `require.Equal(t, c.expected, defaultConfig)`? I think the parameters for Equal are `t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{}` Minor, but could be confusing reading the Error Trace if this test fails.

Reply:
Oh, good one!